### ISSUE TITLE
Issues while installing from source

### ISSUE THREAD

**Issue by akaul** [tmrts/boilr#45]
I'm running into issues while trying to install the package via `go get` when starting from a completely fresh Go workspace.

```
$ go get github.com/tmrts/boilr
src/srcd.works/go-git.v4/repository.go:10:2: use of internal package not allowed
```

I'm using Go 1.8 locally.

```
 $ go version
go version go1.8 darwin/amd64
```

Let me know if there's any additional information I can provide to help debug the issue.

**Comment by rawkode** [comment]
Hi @akaul, this should be fixed by #42

**Comment by akaul** [comment]
Thanks @rawkode! 

I've verified this is working locally, so I'm going to close the issue.
